packages/common/src/client/appeals-api-client.js

```diff
@@ -116,6 +116,16 @@ class AppealsApiClient {
 		return response.json();
 	}
 
+	/**
+	 * @param {{ userId: string, appealSubmissionId: string }} params
+	 * @returns {Promise<(AppealCase|AppealSubmission)>}
```

Comment:
This only returns `AppealSubmission`, right?

Reply:
yep true, will change, thanks

Reply:
Done 👍